modules/rosa-sts-required-roles-and-policies.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// rosa_architecture/rosa-sts-about-iam-resources.adoc
+//
+:_content-type: CONCEPT
+[id="rosa-sts-required-roles-and-policies_{context}"]
+= Configuring ROSA cluster roles and policies
+
+Before creating a ROSA cluster, you must create the required Operator roles and set the correct OpenID Connect (OIDC) provider. You can create these roles by using the `rosa` command-line interface (CLI) tool or the ROSA guided installation in {cluster-manager-url}. You can create these roles manually or by using an automatic process that uses pre-defined names for these roles and polices.
+
+[id="rosa-sts-required-roles-and-policies-automatic_{context}"]
+== Automatic `ocm-role` resource creation
```

Comment:
This heading suggests that the procedure is about creating the `ocm-role` resources, whereas it seems to be about creating the account-wide roles and inline policies, as well as the operator policies. You could update the heading to relate explicitly to the procedure.

Another point to note is that you can create these resources using auto mode or manual mode in the CLI, or using auto or manual mode in OCM. So, we need to distinguish between those paths too.